_pytest/config.py

```diff
@@ -1329,10 +1329,14 @@ def determine_setup(inifile, args, warnfunc=None, rootdir_cmd_arg=None):
     if inifile:
         iniconfig = py.iniconfig.IniConfig(inifile)
         is_cfg_file = str(inifile).endswith('.cfg')
+        # TODO: [pytest] section in *.cfg files is depricated. Need refactoring.
         sections = ['tool:pytest', 'pytest'] if is_cfg_file else ['pytest']
         for section in sections:
             try:
                 inicfg = iniconfig[section]
+                if is_cfg_file and section == 'pytest' and warnfunc:
+                    from _pytest.deprecated import SETUP_CFG_PYTEST
+                    warnfunc('C1', SETUP_CFG_PYTEST.replace('setup.cfg', str(inifile)))
```

Comment:
for consistency, lets turn `SETUP_CFG_PYTEST` into a format string and use something like `CFG_PYTEST_SETION.format(filename=apath)` on all usage sites

Reply:
Fixed